setup.py

```diff
@@ -113,8 +113,8 @@ def run_setup(extension_modules=None):
             "derpconf==0.*,>=0.8.3",
             "libthumbor==2.*,>=2.0.0",
             "opencv-python-headless==4.*,>=4.2.0",
-            "pillow==7.*,>=7.0.0",
-            "pytz==2019.*,>=2019.3.0",
+            "pillow==8.*,>=8.0.0",
```

Comment:
I think we should allow 7 as well

Reply:
Okay, updated. (Should we also allow pytz 2019.3.0?)

Reply:
Yeah, sure. If older version still works and we don’t need anything from new version, there is no reason to restrict.

Reply:
Updated again.